README.md

```diff
@@ -57,11 +57,7 @@ Run the following command  (check help string for more details):
 ```
 
 ### Fetching logs
-Run the following command (check help string for more details):
-
-```shell
-./vse-sync-collection-tools logs --kubeconfig="${KUBECONFIG}"
-```
+The log subcommand has been removed. Instead we have implimented at collector which is enabled by default. If possible you should use a log aggregator.
```

Comment:
If we use a log aggregator, what should we do? disable the collector? if so, how?

Reply:
You can select the collectors that run using the `-s` flag.